chains/orchestrator-relays/runtime/dancelight/src/tests/common/xcm/mocknets.rs

```diff
@@ -104,17 +113,77 @@ decl_test_parachains! {
     }
 }
 
+thread_local! {
+    pub static ETH_BRIDGE_SENT_MSGS: RefCell<Vec<CommittedMessage>> = RefCell::new(Vec::new());
+}
+pub fn eth_bridge_sent_msgs() -> Vec<CommittedMessage> {
+    ETH_BRIDGE_SENT_MSGS.with(|q| (*q.borrow()).clone())
+}
+pub struct DancelightEthMockBridgeHandler;
+impl BridgeMessageHandler for DancelightEthMockBridgeHandler {
+    fn get_source_outbound_messages() -> Vec<BridgeMessage> {
+        // Get messages from the outbound queue
+        let msgs = DancelightRelay::ext_wrapper(|| {
+            snowbridge_pallet_outbound_queue::Messages::<<DancelightRelay as Chain>::Runtime>::get()
+        });
+
+        // Store messages in our static mock buffer
+        ETH_BRIDGE_SENT_MSGS.with(|sent_msgs| {
+            sent_msgs.borrow_mut().extend(msgs.clone());
+        });
+
+        // TODO: return the right Vec<BridgeMessage> here
+        Default::default()
+    }
+
+    fn dispatch_target_inbound_message(
+        _message: BridgeMessage,
+    ) -> Result<(), BridgeMessageDispatchError> {
+        unimplemented!("dispatch_target_inbound_message")
+    }
+
+    fn notify_source_message_delivery(_lane_id: BridgeLaneId) {
+        unimplemented!("notify_source_message_delivery")
+    }
+}
+
+pub struct DancelightEthMockBridge;
+impl Bridge for DancelightEthMockBridge {
+    type Source = DancelightRelay;
+    type Target = ();
+    type Handler = DancelightEthMockBridgeHandler;
+
+    fn init() {
+        <DancelightRelay as Chain>::Network::init();
+    }
+}
+
 decl_test_networks! {
     pub struct DancelightMockNet {
         relay_chain = Dancelight,
         parachains = vec![
             FrontierTemplateDancelight,
             SimpleTemplateDancelight,
         ],
-        bridge = ()
+        bridge = DancelightEthMockBridge
     }
 }
 
+pub fn force_process_bridge() {
+    type DancelightRelayMessageQueue = <DancelightRelay as DancelightRelayPallet>::MessageQueue;
```

Comment:
I would make this more generic. Maybe we can pass a mocknet and adjust the rest of the values? maybe not, just asking for reference